core/iwasm/include/wasm_export.h

```diff
@@ -699,6 +705,10 @@ wasm_runtime_validate_app_addr(wasm_module_inst_t module_inst,
  * space or memory space. Moreover, it checks whether it is the offset of a
  * string that is end with '\0'.
  *
+ * Note: The validation result, especially the NUL termination check,
```

Comment:
Should be NULL?

Reply:
i usually use NUL for '\0' and NULL for (void *)0.